lib/index.js

```diff
@@ -85,6 +86,7 @@ Table.prototype.toString = function (){
     , chars = options.chars
     , truncater = options.truncate
     , colWidths = options.colWidths || new Array(this.head.length)
+    , min_column_width = options.min_column_width || 15
```

Comment:
Thy magic constant has imploded the universe! (Because, magic and stuff). 

On a more serious note, consider describing it, at least in a comment.